dataservice/api/sample_relationship/models.py

```diff
@@ -0,0 +1,165 @@
+from sqlalchemy import event, or_
+
+
+from dataservice.extensions import db
+from dataservice.api.common.model import Base, KfId
+from dataservice.api.sample.models import Sample
+
+
+class SampleRelationship(db.Model, Base):
+    """
+    Represents a relationship between two samples.
+
+    The relationship table represents a tree.
```

Comment:
The idea here - comparing this to family_relationship is that family_relationship is a network while this is _only_ a tree, correct?

Reply:
correct